cloudbuild-provision-cluster.yaml

```diff
@@ -151,14 +151,17 @@ steps:
   args:
   - '-c'
   - |
-      firewall=$(gcloud compute firewall-rules list \
-          --filter="name:k8s-fw AND targetTags.list()~^gke-${_CLUSTER_NAME}-[0-9a-z]*-node$" \
-          --format="csv[no-heading](name)"); \
-      while [[ -z $firewall ]]; do \
+      CLUSTER_CREATE_TIME=$(gcloud container clusters describe ${_CLUSTER_NAME} --zone=${_CLUSTER_LOCATION} --format="value(createTime)")
```

Comment:
Perhaps add an explanatory comment here about why this is necessary

Reply:
Done